src/components/footer/footer.ts

```diff
@@ -16,10 +16,6 @@ export class FooterComponent extends LitElement {
      <footer class="container as-footer">
```

Comment:
https://github.com/AnalogStudiosRI/www.analogstudios.net/pull/60#pullrequestreview-843745890
Reference to html hardcoded in the footer instead of new file.